apps/src/blockly/addons/cdoUtils.js

```diff
@@ -130,28 +130,28 @@ function testJsonSerialization(workspace) {
   const experimentEnabled = experiments.isEnabled(experiments.BLOCKLY_JSON);
   const FIREHOSE_STUDY = 'blockly-json';
   const FIREHOSE_EVENT = 'block-differences';
+  const SORT_BY_POSITION = true;
   Blockly.Events.disable();
 
   // Create an array of blocks based on JSON serialization of the current workspace.
   const jsonSerialization = Blockly.serialization.workspaces.save(workspace);
   const tempJsonWorkspace = new Blockly.Workspace();
   Blockly.serialization.workspaces.load(jsonSerialization, tempJsonWorkspace);
-  const jsonBlocks = tempJsonWorkspace.getAllBlocks();
+  const jsonBlocks = tempJsonWorkspace.getAllBlocks(SORT_BY_POSITION);
 
   // Create an array of blocks based on the XML encoding of the current workspace.
   const xmlSerialization = Blockly.Xml.blockSpaceToDom(workspace);
   const tempXmlWorkspace = new Blockly.Workspace();
   Blockly.Xml.domToWorkspace(xmlSerialization, tempXmlWorkspace);
-  const xmlBlocks = tempXmlWorkspace.getAllBlocks();
+  const xmlBlocks = tempXmlWorkspace.getAllBlocks(SORT_BY_POSITION);
 
   // compareBlockArrays returns an array of differences found.
   const differences = compareBlockArrays(xmlBlocks, jsonBlocks);
   if (differences.length > 0 || experimentEnabled) {
     // Log a record to Firehose/Redshift.
     const recordData = {
-      projectUrl: dashboard.project.getShareUrl(),
```

Comment:
Why did you remove `projectURL`?

Reply:
When looking at the RedShift data, I realized that the url is already included under the `device` column that we automatically collect. It's also a better url than the project share link because it includes things like `/view` or `/edit` or tells us if they were viewing a curriculum level and not in a standalone project. Was just going to add a note to explain!

Reply:
Cool - thanks!